ArchiSteamFarm/Core/Utilities.cs

```diff
@@ -319,6 +325,37 @@ public static class Utilities {
 			}
 		}
 
+		internal static bool IsWeakPassword(string password, params string[] additionallyForbiddenWords) {
+			if (string.IsNullOrEmpty(password)) {
+				throw new ArgumentNullException(nameof(password));
+			}
+
+			if (password.Length < 10) {
+				return true;
+			}
+
+			if (ForbiddenPasswordWords.Any(word => password.Contains(word, StringComparison.InvariantCultureIgnoreCase))) {
```

Comment:
use `OrdinalIgnoreCase` instead of `InvariantCultureIgnoreCase` wherever possible (also in other places).

Reply:
I wanted `InvariantCulture`, because people from countries like Turkey might actually have ASFs locale set to english, but be the password in their native language (some letters are different). Since we have quite a lot of `i`s in there (which is one of the letters), that may ignore stuff like `ARCHISTEAMFARM`.

Reply:
Then it should probably be `CurrentCulture` as I'm not sure if invariant actually covers your case.